README.md

```diff
@@ -7,20 +8,21 @@
 ## Usage
 
 ```js
-var v = require('vec3');
+import { v, Vec3 } from '../src/mod.js'
```

Comment:
That seems wrong. Users of this package don't have src/mod.js

Reply:
shitty copy paste from test suite